README.md

```diff
@@ -17,6 +17,9 @@ brew tap cfergeau/crc
 brew install vfkit
 ```
 
+### Building
+
+From the root direction of this repository, run `make`. You will need a valid Apple Developer certificate to sign the binaries with the required entitlements.
```

Comment:
A valid Apple Developer certificate is not required to sign the binary, the Makefile uses `codesign -s -` aka ad hoc signing ( https://ss64.com/mac/codesign.html ), which works without any Apple certificate.